README.md

```diff
@@ -2,32 +2,30 @@
-  - [outcome](https://github.com/scerioli/Global_Preferences_Survey/tree/master/files/outcome): the csv files containing the data we use for our plots. Hopefully, this is the result one should obtain when using the same input and following the
-  steps of the analysis as we indicate
+  - [income](https://github.com/scerioli/Global_Preferences_Survey/tree/master/files/income): files used for the analysis, modified to be easy to read. These files are the ones that should allow the reader to reproduce in the most complete way the analysis. The core GPS dataset on economic preferences is not included due to copyright restrictions, but can be easily downloaded using the links on the official webpage we provided.
+  - [outcome](https://github.com/scerioli/Global_Preferences_Survey/tree/master/files/outcome): the csv files that containe the results of our replication of the analysis that we subsequently use for plots. The same files can be obtained by running the analysis pipeline
 - [functions](https://github.com/scerioli/Global_Preferences_Survey/tree/master/functions)
-  - It contains all the functions that are very specific for the analysis
-  - [helper_functions](https://github.com/scerioli/Global_Preferences_Survey/tree/master/functions/helper_functions): In this directory, all the helper functions created for the analysis used in the main code and to create the plots can be found. Nothing needs to be done, in the main code they are sourced thanks to the SourceFunctions.r
-- [plots](https://github.com/scerioli/Global_Preferences_Survey/tree/master/plots): The relevant plots are saved here. The plots starting with "main" indicate those plots found in the main article, while the "supplementary" indicate the plots found in the supplementary online material. Two extra plots for the comparison between our analysis and the article's results are there.
-- [GenderPreferencesAnalysis.r](https://github.com/scerioli/Global_Preferences_Survey/blob/master/GenderPreferencesAnalysis.r) is the main code of this analysis, and it produces the outcome files.
-- [CreatePlotsArticle.r](https://github.com/scerioli/Global_Preferences_Survey/blob/master/CreatePlotsArticle.r) uses the outcome files to create the plot of the analysis (both in the main article and in the supplementary online material).
-- [ReproduceArticle.md](https://github.com/scerioli/Global_Preferences_Survey/blob/master/ReproduceArticle.md), as already mentioned, helps (hopefully) the readers to get through the main code and the concept of the analysis. For a more in-depth look at the analysis and the methods, the authors refer to the main paper (mentioned above) and to this file.
+  - The replication of  each figure of the manuscript is represented by function. The folder contains all these functions.
```

Comment:
Not correct: Each function is part of the analysis and preparation of the variables needed for plotting. The helper_functions are usually more general (and reusable) than the functions